tests/nnstreamer_filter_mvncsdk2/unittest_filter_mvncsdk2.cc

```diff
@@ -76,6 +79,22 @@ TEST (pipeline_mvncsdk2_filter, launch_normal)
     EXPECT_EQ (ret, GST_STATE_CHANGE_SUCCESS);
     EXPECT_EQ (state, GST_STATE_NULL);
 
+    /* Check framework auto option */
+    pipeline = replace_string(pipeline, "movidius-ncsdk2", "auto", NULL, &changed);
+    EXPECT_EQ(changed, 1U);
+
+    gstpipe = gst_parse_launch (pipeline, &err);
+    EXPECT_TRUE(gstpipe != NULL);
```

Comment:
minor style issues: how about add white spaces for style consistency in three lines: 
replace_string (...), EXPECT_EQ (...), EXPECT_TRUE (...)

Reply:
Oh.. thank you. Updated.